logging/api/logging.api

```diff
@@ -44,6 +44,10 @@ public final class com/juul/tuulbox/logging/Log {
 	public static synthetic fun warn$default (Lcom/juul/tuulbox/logging/Log;Ljava/lang/Throwable;Ljava/lang/String;Lkotlin/jvm/functions/Function0;ILjava/lang/Object;)V
 }
 
+public synthetic class com/juul/tuulbox/logging/LogAtomicTagGeneratorRefVolatile {
+	public fun <init> (Ljava/lang/Object;)V
+}
```

Comment:
Can't quite pin down where this is coming from (and if it needs to be on the public API)?

Reply:
It's auto-generated from atomicfu's plugin. Not totally sure why

Reply:
I think the `synthetic` marker prevents library consumers from accessing it directly?

Reply:
> It's auto-generated from atomicfu's plugin. Not totally sure why

Oh gotchya. If it's generated from atomicfu, not much we can do (seems fine) — I was just a little confused where it was coming from.

> I think the `synthetic` marker prevents library consumers from accessing it directly?

Not sure actually.